index.js

```diff
@@ -58,7 +58,7 @@
 
       var fn = this.__outsideClickHandler = (function(localNode, eventHandler) {
         return function(evt) {
-          evt.stopPropagation();
+          evt.stopImmediatePropagation();
```

Comment:
better to do this as check + fn call, preserving stopPropagation, too:
```
if (evt.stopImmediatePropagation) {
  evt.stopImmediatePropagation();
} else {
  evt.stopPropagation();
}
```
because browser...